### ISSUE TITLE
[Bug] CollectionView Layout issues

### ISSUE THREAD

**Issue by daves1992** [xamarin/Xamarin.Forms#12722]
I noticed since switching to CollectionView from ListView there is multiple layout issues

These issues are apparent when the screen first loads but disappear when you scroll.

XAML:

```
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="MaceXamarinSharedStandardProject.Views.SelectionPage"
             x:Name="CPSelectionPage"
             Title ="{Binding Title}"
             Style="{StaticResource contentPage}">
    <ContentPage.Content>
        <StackLayout Orientation="Vertical" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand" Padding="0,10,0,0">
            <Entry Text="{Binding FilterText}" IsVisible="{Binding ShowFilter}" Placeholder="Filter" />
            <CollectionView x:Name="lstSelection" SelectionChangedCommand="{Binding SelectionChangedCommand}" SelectedItem="{Binding SelectedItem}" ItemsSource="{Binding SelectionList}" VerticalOptions="FillAndExpand" SelectionMode="Single" ItemSizingStrategy="MeasureAllItems">
                <CollectionView.ItemTemplate>
                    <DataTemplate>
                        <Frame Style="{StaticResource borderFrame}" Padding="1" HasShadow="False" CornerRadius="0" >
                            <Frame Style="{StaticResource backgroundFrame}" Padding="10" HasShadow="False" CornerRadius="0">
                                <StackLayout Orientation="Vertical" >
                                    <Label Text="{Binding ID}" FontAttributes="Bold" Style="{StaticResource textLabel}" />
                                    <Label Text="{Binding Description}" FontSize="Small" Style="{StaticResource detailLabel}" />
                                </StackLayout>
                            </Frame>
                        </Frame>
                    </DataTemplate>
                </CollectionView.ItemTemplate>
            </CollectionView>
        </StackLayout>
    </ContentPage.Content>
</ContentPage>
```

Issue:
![CollectionView](https://user-images.githubusercontent.com/29727003/98009584-9ccbf900-1ded-11eb-82dc-9aecbad4211c.gif)

- Version with issue: 4.8.0.1560
- Last known good version: Unknown
- IDE: VS 4.8.04084

- Affected Devices:
Seems to only affect IOS

**Comment by rmarinho** [comment]
Hi @daves1992  thanks for your report, is it possible to upload a small repro of the issue so we can investigate better the problem?!

Thanks

**Comment by ClaytonIndustries** [comment]
@rmarinho You can repro this easily by creating a collection view where the items are different heights. `ItemSizingStrategy="MeasureAllItems"` doesn't seem to work properly on iOS, pretty sure this issue has been raised many times before

**Comment by rmarinho** [comment]
Tested with 4.8.0.1560 and 5.0.0-pre3
Seems i can reproduce this 

on load:
![image](https://user-images.githubusercontent.com/1235097/98284531-2966ea80-1f99-11eb-918d-154a0fd1ad51.png)

after i scroll :
![image](https://user-images.githubusercontent.com/1235097/98284660-59ae8900-1f99-11eb-80d6-08a33779b020.png)

**Comment by hartez** [comment]
This is fixed in 5.0.0.1931.
